### PR TITLE
external/libwally-core: update to v0.8.9

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: src/keystore.c [hunk 1]
```diff
@@ -784,7 +784,7 @@ bool keystore_secp256k1_schnorr_bip86_sign(
     const secp256k1_context* ctx = wally_get_secp_context();
     uint8_t aux_rand[32] = {0};
     random_32_bytes(aux_rand);
-    if (secp256k1_schnorrsig_sign(ctx, sig64_out, msg32, &keypair, aux_rand) != 1) {
+    if (secp256k1_schnorrsig_sign32(ctx, sig64_out, msg32, &keypair, aux_rand) != 1) {
```
Comment (Beerosagos) [inline src/keystore.c:787]: Is this meant to be here? My compiler complains about not finding `secp256k1_schnorrsig_sign32` declaration

Comment (benma) [reply src/keystore.c:787]: Yeah. CI passes too. Did you update the submodule, which now points to another commit?

Comment (Beerosagos) [reply src/keystore.c:787]: Yikes, forgot that! Thanks

**Review by Beerosagos — APPROVED** [review]
tACK :+1:

**Pull request merged**